vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthResetPasswordFragment.kt

```diff
@@ -120,24 +115,10 @@ class FtueAuthResetPasswordFragment @Inject constructor() : AbstractFtueAuthFrag
     }
 
     override fun updateWithState(state: OnboardingViewState) {
-<<<<<<< HEAD
-//        setupUi(state)
-
-        when (state.asyncResetPassword) {
-            is Loading -> {
-                // Ensure new password is hidden
-                views.passwordField.hidePassword()
-            }
-            is Fail    -> {
-                views.resetPasswordEmailTil.error = errorFormatter.toHumanReadable(state.asyncResetPassword.error)
-            }
-            is Success -> Unit
-=======
         setupUi(state)
         if (state.isLoading) {
             // Ensure new password is hidden
             views.passwordField.hidePassword()
->>>>>>> v1.4.11
```

Comment:
I don't understand why we have conflicts in this file. it seems we don't use it in Tchap, right ?